test/integration/browser/src/index.ts

```diff
@@ -35,11 +35,6 @@ type BrowserType = 'chromium' | 'firefox' | 'webkit';
 async function runTestsInBrowser(browserType: BrowserType, endpoint: url.UrlWithStringQuery, server: cp.ChildProcess): Promise<void> {
 	const browser = await playwright[browserType].launch({ headless: !Boolean(optimist.argv.debug) });
 	const context = await browser.newContext();
-	try {
-		await context.grantPermissions([`clipboard-read`, `clipboard-write`]); // some tests need clipboard access
```

Comment:
Is the permission not needed any more ?

Reply:
@deepak1556 playwright does not seem to support this, I filed https://github.com/microsoft/playwright/issues/19888

Reply:
I think with https://chromium-review.googlesource.com/c/chromium/src/+/1895749, there is only `clipboard-write` with `allowWithoutSanitization: <Boolean>` property set. I wonder if playwright needs to update the permission name.

```
clipboard-read-write -> clipboard-write with allowWithoutSanitization: true
clipboard-sanitized-write -> clipboard-write with allowWithoutSanitization: false
```

Reply:
@deepak1556 oh nice findings, maybe tell them in the issue?